### PR TITLE
Set up docker to use linter

### PR DESCRIPTION
**Pull request by warku123**
Set up docker to use linter.
I use docker to set up a new linter env and install all necessary python packages.

But now it looks can't work.

Need to update.

### CODE REVIEW

**Comment by warku123** [comment]
I fix few error by using flake8 instead of Pylint. Because of the pylint import error is a little bit annoying with the environment, even though I've install the env all right.

Can see the bug info [here](https://github.com/VirtwhoQE/hypervisor-builder/pull/30/checks).

I also neglect a few type of error like format errors, you can see them in `.flake8` file

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E501,W605,F632,W503,F541
```
Comment (hkx303) [inline .flake8]: no new line :)

File: .github/workflows/mega-linter.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@ jobs:
   build:
     name: MegaLinter
     runs-on: ubuntu-latest
+    container: quay.io/virtwho-qe/hypervisor-builder-linter:v0
```
Comment (hkx303) [inline .github/workflows/mega-linter.yml:26]: Goot to see our own container :)

File: linter-docker/dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM oxsecurity/megalinter:v6
+
+LABEL maintainer="Pinge Zhang <pinzhang@redhat.com>"
+
+RUN pip3 install paramiko==2.7.2 pytest==6.2.2
```
Comment (hkx303) [inline linter-docker/dockerfile]: new line :)

**Review by hkx303 — APPROVED** [review]
ACK, pending very small comments

**Review by hkx303 — APPROVED** [review]
ACK

**Comment by hkx303** [comment]
Good job :+1:

**Pull request merged**